pkg/process/namespace.go

```diff
@@ -10,6 +10,50 @@ const (
 	AnnotationNamespaced = MetadataPrefix + "/namespaced"
 )
 
+// This is a list of "cluster-wide" resources harvested from `kubectl api-resources --namespaced=false`
+// This helps us to know which objects we should NOT apply namespaces to automatically.
+// We can add to this list periodically if new types are added. There is no reason not to add popular CRD types here as well.
+// Alternatively, library authors can add annotations to control namespacing for a type as well.
+var clusterWideKinds = []string{
```

Comment:
Just make this a string bool map and you can avoid the function :)

Reply:
grrrr... ok.